scripts/build_nim.sh

```diff
@@ -90,6 +92,23 @@ nim_needs_rebuilding() {
 		# NIM_COMMIT is empty, so assume the commit we need is already checked out
 		NIM_COMMIT_HASH="$(git rev-list -n 1 HEAD)"
 	fi
+
+	if [[ ! -d "$NIMBLE_DIR" ]] && [[ -n "$NIMBLE_COMMIT" ]]; then
```

Comment:
When changing `NIMBLE_COMMIT` to a new value, it appears to me that `NIMBLE_DIR` will stick to the stale version that's already in it from a prior run.